larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -150,22 +150,28 @@ TfLiteStatus Prepare(KernelType kernel_type,
   // If an error happened in Init, then report an error message
   if (!conv_params->conv_params_initialized) return kTfLiteError;
 
-  TF_LITE_ENSURE(context, node->inputs->size == 4);
+  TF_LITE_ENSURE(context, node->inputs->size == 5);
 
   const auto* input = GetInput(context, node, 0);
   const auto* filter = GetInput(context, node, 1);
   const auto* post_activation_multiplier = GetInput(context, node, 2);
   const auto* post_activation_bias = GetInput(context, node, 3);
+  const auto* thresholds = GetInput(context, node, 4);
```

Comment:
What is the benefit of passing the thresholds as an operand rather than a kernel attribute?

Currently the threshold will always be a constant that is computed during the conversion process and cannot be the output of an other op unless we change the pattern to compute the threshold using TFLite ops. These ops would then be constant folded in the normal case but could be part of the graph if multiplier and bias are dynamic. However this is probably not practical due to all the edge cases we handle in the current conversion.

Reply:
Yeah you're right, it probably could be an attribute rather than an operand, I hadn't thought about that. How strongly do you feel about this?

Reply:
I don't feel strongly either way, I might even prefer it being an operand. I was just curious about the reasoning.
This came up when looking at the attribute name and I got confused that this PR didn't mention `write_bitpacked_threshold` before was #387 merged and the unittests still pass...

Reply:
I don't feel strongly about it. So far I had been using the idea that variable-length data (i.e. depending on input shape) would be input and constant-size data would be attributes. But I'm fine with changing it if that makes more sense.

Reply:
> 

I don't feel strongly about it. So far I had been using the idea that variable-length data (i.e. depending on input shape) would be input and constant-size data would be attributes. But I'm fine with changing it if that makes more sense.

I agree, I think it makes sense to keep it an operand 👍